### PR TITLE
Modify test, cancel allocation is independant of challenges

### PR DESCRIPTION
**Pull request by Sriep**
To work with 0chain PR https://github.com/0chain/0chain/pull/637. 
`cancelAllocation` no longer gives an error if not enough challenges have passed.

Passed test https://github.com/0chain/system_test/runs/4092818034?check_suite_focus=true

### CODE REVIEW

File: tests/cli_tests/zboxcli_update_allocation_test.go [hunk 1]
```diff
@@ -179,18 +179,13 @@ func TestUpdateAllocation(t *testing.T) {
...
 
-		output, err := cancelAllocation(t, configPath, allocationID)
-
-		require.NotNil(t, err, "expected error when canceling allocation", strings.Join(output, "\n"))
-		require.True(t, len(output) > 0, "expected output length be at least 1", strings.Join(output, "\n"))
-		// FIXME; note incorrect "error creating" error message: should be "error updating"
-		require.Equal(t, "Error creating allocation:[txn] too less sharders to "+
-			"confirm it: min_confirmation is 50%, but got 0/2 sharders", output[0])
+		_, err := cancelAllocation(t, configPath, allocationID)
+		require.NoError(t, err, "cancel allocation failed but should succeed")
```
Comment (stewartie4) [inline tests/cli_tests/zboxcli_update_allocation_test.go:188]: assign _ on line 187 to variable called output
add Strings.join(output, "\n") as another String param to require.NoError
that will give good info if there is an error

**Comment by Sriep** [comment]
Passed system test with 0chianPR https://github.com/0chain/0chain/pull/637  https://github.com/0chain/system_test/runs/4105276072?check_suite_focus=true

**Comment by stewartie4** [comment]
thanks, we can merge this once your change merges to staging

**Comment by Sriep** [comment]
https://github.com/0chain/0chain/pull/637 has been merged

File: tests/cli_tests/zboxcli_update_allocation_test.go [hunk 2]
```diff
@@ -179,18 +179,13 @@ func TestUpdateAllocation(t *testing.T) {
 		)
 	})
 
-	t.Run("Cancel Allocation Should fail when blobber passes challenges", func(t *testing.T) {
+	t.Run("Cancel_allocation_immediately_should_succeed", func(t *testing.T) {
```
Comment (stewartie4) [inline tests/cli_tests/zboxcli_update_allocation_test.go:182]: There are already a tests for this in 
unhealthy_blobber_test.go ->
"cancel Allocation Should Work when blobber fails challenges"
"cancel Other's Allocation Should Fail"
You could just move these in here now that they no longer require a bad blobber

**Review by stewartie4 — CHANGES_REQUESTED** [review]
move existing test - see above

**Comment by stewartie4** [comment]
@Sriep thanks, happy to merge once staging has the code fix

**Pull request merged**